### DISCUSSION TITLE
Vercel cron job

### DISCUSSION THREAD

**Discussion by christojacques** [vercel/community#3121] [Help]
### Question

In node js when i am adding cron jobs it is showing on vercel but when i run them it is giving an error of not found on this path
can anyone guide what is wrong with my path?
![image](https://github.com/vercel/community/assets/137637546/6f7fab56-0302-4fed-8120-0bf24cbf403d)

**Comment by amyegan** [comment]
Hi @christojacques. The `path` should relate to a serverless function (or edge function). So if the serverless function were deployed to `example.com/api/cron-function`, then the `path` should be set to `/api/cron-function`
